akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/EventSourcedBehavior.scala

```diff
@@ -232,4 +232,10 @@ object EventSourcedBehavior {
    */
   @InternalApi
   private[akka] def withReplication(context: ReplicationContextImpl): EventSourcedBehavior[Command, Event, State]
+
+  /**
+   * Override to define a custom stash capacity per entity.
+   * If not defined, the default `akka.persistence.typed.stash-capacity` will be used.
+   */
```

Comment:
This trait is marked with `@DoNotInherit`. Users of the scaladsl don't extend this and therefore won't have the change to override this method. Instead, they get an instance of `EventSourcedBehaviorImpl` to configure further.